test/tall/regression/0500/0506.unit

```diff
@@ -65,14 +65,15 @@ class Foo {
 <<<
     main() {
       tree.Node definition = new tree.FunctionExpression(
-          makeIdentifier(param.name),
-          makeParameters(param.parameters),
-          null, // body
-          param.type == null ? null : makeType(param.type),
-          makeEmptyModifiers(), // TODO: Function parameter modifiers?
-          null, // initializers
-          null, // get/set
-          null); // async modifier
+        makeIdentifier(param.name),
+        makeParameters(param.parameters),
+        null, // body
+        param.type == null ? null : makeType(param.type),
+        makeEmptyModifiers(), // TODO: Function parameter modifiers?
+        null, // initializers
+        null, // get/set
+        null,
+      ); // async modifier
```

Comment:
I think it's probably fine to expect authors to manually move some edge case comments like this.


Should we move it here too, or leave it as is and let the regression test remain non-representative for code using the new format?

Reply:
My rule for regression tests is to always leave them as-is. (That's why, for example, many of them still contain `new` even though it's no longer idiomatic to use it.)